.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,48 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/docker-existing-dockerfile
+{
+    "name": "Codespace setup",
+    "build": { "dockerfile": "Dockerfile" },
+    // Features to add to the dev container. More info: https://containers.dev/features.
+    "features": {
+        // Enable running docker in docker
+        "ghcr.io/devcontainers/features/docker-in-docker:2": {},
+        "ghcr.io/devcontainers/features/sshd:1": {},
+        "ghcr.io/devcontainers/features/github-cli:1": {},
+        "ghcr.io/devcontainers/features/common-utils:2": {},
+        "ghcr.io/devcontainers/features/python:1": {},
+        "ghcr.io/devcontainers-contrib/features/shfmt:1": {},
+        "ghcr.io/devcontainers/features/terraform:1": {}
+    },
+    "customizations": {
+        "vscode": {
+            "settings": {
+                "files.watcherExclude": {
+                    "**/target": true,
+                    "**/.bloop": true,
+                    "**/node_modules": true,
+                    "**/__pycache__": true
+                },
+                "search.exclude": {
+                    "*/target": true,
+                    "**/.bloop": true,
+                    "**/node_modules": true,
+                    "**/__pycache__": true
+                },
```

Comment:
(question) Are there any terraform directories to ignore?

Reply:
I actually realized I didn't need this at all since we just have the config files in this directory and don't build anything here